content/2.developers/2.developer-tools/4.arch3js/3.tutorials/4.websockets.md

```diff
@@ -188,7 +188,7 @@ Now, you can check how to subscribe to the <a href="https://docs.tendermint.com/
 
 ### Finding events
 
-The easiest way to find the list of events linked to a particular transaction is to check the events logs within a block exporer. These event logs will also show fields that can be used as additional filters. For example the following transaction https://www.mintscan.io/archway/tx/A2EFABA14F892C0E559205F5B0B3FB0D1F9533E086BEDAF513954EE2EAC63F69?height=1628545 has the **coin_received** and you can also use the **receiver** field as a filter. The final filter would be **tm.event = 'Tx' AND coin_received.receiver = ''**.
+The easiest way to find the list of events linked to a particular transaction is to check the events logs within a block explorer. These event logs will also show fields that can be used as additional filters. For example the following transaction https://www.mintscan.io/archway/tx/A2EFABA14F892C0E559205F5B0B3FB0D1F9533E086BEDAF513954EE2EAC63F69?height=1628545 has the **coin_received** and you can also use the **receiver** field as a filter. The final filter would be **tm.event = 'Tx' AND coin_received.receiver = ''**.
```

Comment:
The easiest way to find the list of events linked to a particular transaction is to check the events logs within a block explorer. These event logs will also show fields that can be used as filters. For example the following transaction https://www.mintscan.io/archway/tx/A2EFABA14F892C0E559205F5B0B3FB0D1F9533E086BEDAF513954EE2EAC63F69?height=1628545 has the **coin_received** event, which has a **receiver** field that you can use as a filter.

The following filter will listen to any blocks which include a **coin_received.receiver** event:

**tm.event = 'Tx' AND transfer.recipient EXISTS**

And this one will listen to blocks where the event include a specific address in **coin_received.receiver**:

**tm.event = 'Tx' AND transfer.recipient CONTAINS 'archway1du5canv32dd7qsp5kz470ancmcn6r6650qdn9z7wp9ke7wnhpycqztx9js'**